Libraries/acdriver/Headers/acdriver/Compile/Output.h

```diff
@@ -138,6 +162,12 @@ class Output {
      * The identifier of an asset for use in results.
      */
     static std::string AssetReference(xcassets::Asset::Asset const *asset);
+
+public:
+    /*
+     * Translates a type extension of an acceptable extra type to ExtraImageType enum.
+     */
+    static ext::optional<ExtraImageType> ExtraImageTypeFromExtension(std::string const &extension);
```

Comment:
This function should live in the ExtraImageType class.

Reply:
ExtraImageType class was a enum class, it can not define functions... :S